app/ui/browser/lib/user-agent.js

```diff
@@ -78,3 +109,13 @@ export function createHistory({ url, title, session }) {
   });
 }
 
+/**
+ * Unauthenticated Requests
```

Comment:
_Authenticated_ is the wrong term here.  Do you mean _ambient_ or something similar?